src/adapters/adbund.js

```diff
@@ -0,0 +1,81 @@
+var utils = require('../utils.js');
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+
+var adBundAdapter = function adBundAdapter() {
+  var timezone = (new Date()).getTimezoneOffset();
+  var bidAPIs = [
+    'http://us-east-engine.adbund.xyz/prebid/ad/get',
+    'http://us-west-engine.adbund.xyz/prebid/ad/get'
+  ];
+  //根据时区来选择接口服务器
+  var bidAPI = bidAPIs[timezone < 0 ? 0 : 1];
+  //bidAPI = 'http://52.66.158.121:8888/prebid/ad/get';
+
+  function _stringify(param) {
+    var result = [];
+    var key;
+    for (key in param) {
+      if (param.hasOwnProperty(key)) {
+        result.push(key + '=' + encodeURIComponent(param[key]));
+      }
+    }
+    return result.join('&');
+  }
+
+  function _jsonp(server, param, handler) {
+    var head = document.getElementsByTagName('head')[0];
+    var script = document.createElement('script');
+    var callbackName = 'jsonp_' + (new Date()).getTime().toString(36);
+
+    param[param.jsonp] = callbackName;
+    window[callbackName] = function (data) {
```

Comment:
Add your callback function to $$PREBID_GLOBAL$$ object instead of adding it to window object